builder/azure/arm/builder.go

```diff
@@ -177,7 +177,8 @@ func (b *Builder) Run(ui packer.Ui, hook packer.Hook, cache packer.Cache) (packe
 			NewStepDeployTemplate(azureClient, ui, b.config, deploymentName, GetVirtualMachineDeployment),
 			NewStepGetIPAddress(azureClient, ui, endpointConnectType),
 			&StepSaveWinRMPassword{
-				Password: b.config.tmpAdminPassword,
+				Password:  b.config.tmpAdminPassword,
```

Comment:
Did the spacing rules get updated?  I am surprised this wasn't already spaced correctly.

Reply:
it automatically re-spaces when new fields are added to the struct to make them line up.

Reply:
Now I see it.  Thank you!